utils/bluez-tools/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2021 Karl Osterseher <karli_o@gmx.at>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := bluez-tools
+PKG_VERSION := 0.1.38-662e
+PKG_RELEASE := $(PKG_SOURCE_VERSION)
+
+PKG_MAINTAINER := Alex Orlenko <none>
+PKG_LICENSE := GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE_PROTO := git
+PKG_SOURCE_URL := https://github.com/khvzak/bluez-tools.git
+PKG_SOURCE_VERSION := f653217
+PKG_BUILD_DIR := $(BUILD_DIR)/bluez-tools-$(PKG_VERSION)
```

Comment:
Please remove the whitespaces before `:=` and after.
So that it looks consistent throughout the file

Reply:
done